source/MaterialXGenOgsXml/OgsFragment.cpp

```diff
@@ -0,0 +1,353 @@
+#include "OgsFragment.h"
+
+#include <MaterialXFormat/XmlIo.h>
+#include <MaterialXGenShader/Util.h>
+#include <MaterialXGenShader/DefaultColorManagementSystem.h>
+#include <MaterialXGenOgsXml/GlslFragmentGenerator.h>
+#include <MaterialXGenShader/GenContext.h>
+#include <MaterialXGenOgsXml/OgsXmlGenerator.h>
+
+#ifdef MATERIALX_BUILD_CROSS
+#include <MaterialXCross/Cross.h>
+#endif
+
+#include <iostream>
+
+namespace MaterialXMaya
```

Comment:
I didn't bother moving namespaces considering that we're likely to move again soon.